TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -46,10 +48,14 @@ static class SimpleDecodedHttpRequestHandler extends ChannelInboundHandlerAdapte
         public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception {
             if (msg instanceof HttpRequest) {
                 currentRequest = (HttpRequest) msg;
-            } else if (msg instanceof LastHttpContent) {
-                isDone = true;
+            } else if (msg instanceof HttpContent) {
+                ((HttpContent)msg).release();
```

Comment:
msg is an Object.  release() is defined on a ReferenceCounted interface.  There's a helper call ReferenceCountUtil that can take an Object and do the dynamic checks, but since I need to do a very similar check already, this seemed more straightforward.